### PR TITLE
Docs: Fix broken image link

### PR DESCRIPTION
**Pull request by oddlittlebird**
Fixing a broken image link

### CODE REVIEW

**Review by kminehart — APPROVED** [review]

File: docs/sources/whatsnew/whats-new-in-v7-5.md [hunk 1]
```diff
@@ -36,13 +36,14 @@ A new Loki logs browser lets you construct the queries step by step: you choose
...
->**Note:** This is not going to affect provisioned data sources or already created data sources.
+
+> **Note:** This is not going to affect provisioned data sources or already created data sources.
 
 ### Word highlighting for Elasticsearch
 
 When searching for text in Elasticsearch logs, matching words in the log line returned by the query are now highlighted.
 
-![Elastic logs highlighting](/img/docs/elastic/elastic-word-highlighting-7-5.png)
+![Elastic logs highlighting](/img/docs/elasticsearch/elastic-word-highlighting-7-5.png)
```
Comment (wbrowne) [inline docs/sources/whatsnew/whats-new-in-v7-5.md:46]: Is the loki one on line 30 okay too?

**Comment by oddlittlebird** [comment]
Y'all are making me paranoid. I'm going to rerun the local build and check again.

**Comment by oddlittlebird** [comment]
Yes, just confirmed again on my local build. Images are present and looking good.

File: docs/sources/whatsnew/whats-new-in-v7-5.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/whatsnew/whats-new-in-v7-5.md:46]: Yes

**Pull request merged**